stable/phabricator/Chart.yaml

```
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: phabricator
-version: 4.3.3
+version: 4.3.4
```

as you are changing the major version for the dependency you should also bump the major version here

```suggestion
version: 5.0.0
```